addons/sale/data/mail_data.xml

```diff
@@ -141,10 +141,15 @@
     <div style="margin: 0px; padding: 0px;">
         <table width="100%" style="color: #454748; font-size: 12px; border-collapse: collapse;">
             <tr style="border-bottom: 2px solid #dee2e6;">
-                <td width="18%"><strong>Products</strong></td>
+                <td style="width: 150px;"><strong>Products</strong></td>
                 <td></td>
-                <td><strong>Quantity</strong></td>
-                <td width="10%" align="center"><strong>Price</strong></td>
+                <td width="15%" align="center"><strong>Quantity</strong></td>
+                % if object.user_id.has_group('account.group_show_line_subtotals_tax_excluded'):
+                    <td width="15%" align="right"><strong>VAT Excl.</strong></td>
+                % endif
+                % if object.user_id.has_group('account.group_show_line_subtotals_tax_included'):
+                    <td width="15%" align="right"><strong>VAT Incl.</strong></td>
+                % endif
```

Comment:
this would still create problems when editing the template, try to keep all modifications just to the content leaving tags untouched

```
                <td width="15%" align="right"><strong>
                    % if object.user_id.has_group('account.group_show_line_subtotals_tax_excluded'):
                        VAT Excl.
                    % else
                        VAT Incl.
                    % endif
                </strong></td>
```

Reply:
Done :+1: